<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListAnggota</class>
 <widget class="QDialog" name="ListAnggota">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anggota Perpustakaan</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="daftarAnggota">
      <attribute name="title">
       <string>Daftar Anggota</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="cariAnggota">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Masukkan kata kunci berdasarkan Nomor Induk / Nama anggota&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="placeholderText">
            <string>Cari Nomor / Nama Anggota </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCari">
           <property name="text">
            <string>Cari !</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tblAnggota"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tambahAnggota">
      <attribute name="title">
       <string>Tambah Anggota</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="bAgtSimpan">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>F5. Simpan</string>
           </property>
           <property name="shortcut">
            <string>F5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAgtBtl">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>F12. Batal</string>
           </property>
           <property name="shortcut">
            <string>F12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QLineEdit" name="lAgtNama"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>No Anggota</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lAgtNo"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lAgtKelas"/>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lAgtJurusan"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nama Anggota</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Kelas</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Jurusan</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lAgtNo</tabstop>
  <tabstop>lAgtNama</tabstop>
  <tabstop>lAgtKelas</tabstop>
  <tabstop>lAgtJurusan</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cariAnggota</sender>
   <signal>returnPressed()</signal>
   <receiver>btnCari</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
